### PR TITLE
Add Setup and other minor updates for a release

### PR DESCRIPTION
**Pull request by pombredanne**
@hughsie for your review!
This is just enough to get a wheel publishable on Pypi

### CODE REVIEW

**Comment by pombredanne** [comment]
Note that I cannot enable Appveyor (windows) or Travis (Linux and mac) CI: only you can afaik

File: test.py [hunk 1]
```diff
@@ -18,49 +18,50 @@
...
 import time
 import hashlib
 
-def check_archive(filename):
+
+def check_archive(filename, expected_rc=0):
     argv = ['cabextract', '--test', '/tmp/test.cab']
     rc = subprocess.call(argv)
-    assert rc == 0
+    assert rc == expected_rc, "inavlid return code: %r" % rc
```
Comment (hughsie) [inline test.py]: invalid

**Comment by hughsie** [comment]
travis enabled

File: test.py (hunk 1 above)
Comment (pombredanne) [reply test.py]: Fixed in dc9a2c2f7f531ea3051d7a37f8fc91ff95f21655

**Comment by pombredanne** [comment]
And we have the CI passing the tests now.

**Comment by hughsie** [comment]
Can you rebase and merge the fixup commits please? If you force-push the branch it'll update the PR.

**Comment by pombredanne** [comment]
Your wish is my command!

I also removed for now the Appveyor config. The would not pass anyway on Windows with the current dep on cabextract

**Comment by pombredanne** [comment]
Thanks! ... It looks like neither of us has a life ;)

**Comment by hughsie** [comment]
I'm stuck in bed with a chest infection :(

**Pull request merged**